// errorcheck

package main

func main() {
	append = 30 // ERROR `use of builtin append not in function call`
}
